{{ cookiecutter.__package_name_kebab_case }}/.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "{{ cookiecutter.__package_name_kebab_case }}",
     "dockerComposeFile": "../docker-compose.yml",
-    "service": "dev",
+    "service": "devcontainer",
```

Comment:
Does this change mean that on VS Code we'll use the stripped down `devcontainer` profile as well? Wouldn't that break e.g. the `~/.gitconfig` mount?

Reply:
Yes, that's right, VS Code (and any other Dev Container compatible tooling such as GitHub Codespaces or the Dev Container CLI) will use the `devcontainer` service.

With regard to the `~/.gitconfig` file, I checked and Dev Containers add those conveniences for us automatically. It's only if we don't use Dev Containers but vanilla Docker (Compose) that we need to take care of the things that Dev Containers normally takes care of (like `.gitconfig`, but also forwarding the SSH agent).